frontend/src/pages/projects/screens/spawner/SpawnerPage.tsx

```diff
@@ -335,9 +335,9 @@ const SpawnerPage: React.FC<SpawnerPageProps> = ({ existingNotebook }) => {
               />
             ) : (
               <FormSection
-                title={SpawnerPageSectionTitles[SpawnerPageSectionID.DATA_CONNECTIONS]}
-                id={SpawnerPageSectionID.DATA_CONNECTIONS}
-                aria-label={SpawnerPageSectionTitles[SpawnerPageSectionID.DATA_CONNECTIONS]}
+                title={SpawnerPageSectionTitles[SpawnerPageSectionID.CONNECTIONS]}
+                id={SpawnerPageSectionID.CONNECTIONS}
+                aria-label={SpawnerPageSectionTitles[SpawnerPageSectionID.CONNECTIONS]}
```

Comment:
I think the jump link should be hidden behind the feature flag like `Connections` instead of changing `Data Connections` to `Connections`. The change here is leading to inconsistencies when the feature flag `disableConnectionTypes=true`, for example, in the screenshot below when `disableConnectionTypes=true`, the form field is pointing to `Data Connections` (Use a data connection checkbox) while the jump links and from section is pointing to `Connections` which is a different field altogether.
<img width="1064" alt="Screenshot 2024-12-10 at 5 38 15 PM" src="https://github.com/user-attachments/assets/6672edc2-3ba5-4794-8506-9a421f08181b">

Reply:
As we're starting to clean up code from data connections, I have removed this code block instead of putting it behind the feature flag.